src/components/layout/RightSectionWallet.vue

```diff
@@ -21,25 +21,50 @@
         {{ address }}
       </div>
       <div class="not-bootstrap-row">
-        <AeAmount :amount="balance" />
+        <Dropdown
+          v-if="hasContractV2Address"
+          :options="tokenBalancesOptions"
+          :method="updateTokenBalance"
```

Comment:
I can't find that it actually changes except the Dropdown value.

Reply:
Yeah, by design this dropdown is just displaying the token balance. It is about to be transformed into a search component in near future.